src/sagemaker/fw_utils.py

```diff
@@ -617,9 +617,9 @@ def warn_if_parameter_server_with_multi_gpu(training_instance_type, distribution
         return
 
     is_multi_gpu_instance = (
-        training_instance_type.split(".")[1].startswith("p")
-        and training_instance_type not in SINGLE_GPU_INSTANCE_TYPES
-    )
+        training_instance_type == "local_gpu"
```

Comment:
how are we checking if its a multi_gpu instance just by checking for "local_gpu"
or is it assumed that the person who is testing locally has verified that his local instance supports multi_gpu

Reply:
We cannot check that, so I updated the warning message to "if"